stdlib/experimental/http/http_experimental.go

```diff
@@ -94,25 +114,38 @@ var get = values.NewFunction(
 			body, err := ioutil.ReadAll(limitedReader)
 			_ = response.Body.Close()
 			if err != nil {
-				return 0, nil, err
+				return 0, nil, nil, err
 			}
 			s.LogFields(
 				log.Int("statusCode", response.StatusCode),
 				log.Int("responseSize", len(body)),
 			)
-			return response.StatusCode, body, nil
-
+			return response.StatusCode, body, HeaderToArray(response.Header), nil
 		}(req)
 		if err != nil {
 			return nil, err
 		}
 
-		return values.NewObjectWithValues(map[string]values.Value{"statusCode": values.NewInt(int64(statusCode)), "body": values.NewBytes(body)}), nil
+		return values.NewObjectWithValues(map[string]values.Value{
+			"statusCode": values.NewInt(int64(statusCode)),
+			"headers":    headers,
+			"body":       values.NewBytes(body)}), nil
 
 	},
 	true, // get has side-effects
 )
 
+func HeaderToArray(header http.Header) (headerObj values.Object) {
```

Comment:
nit, can you rename this function to `headerToObject`? Using lower case means its not a public function.

Reply:
Done!